### ISSUE TITLE
file download --keep-zip doesn't work in windows version

### ISSUE THREAD

**Issue by ptornhult** [lokalise/lokalise-cli-2-go#23]
When running` lokalise2 file download` using `--unzip-to` the ZIP file is removed in linux/mac but remain on windows. Adding the flag `--keep-zip` doesn't do anything in this case either

(We're also using `--format xml` if that helps)

**Comment by ptornhult** [comment]
Might be that we're using the flag incorrectly, right now I've tried with `--keep-zip false` since the docs don't really tell me what to put in the argument if I want the file to be removed after unzipping?

**Comment by beinarovic** [comment]
Hello, and sorry for the late response.

Yes, the flag format for booleans is different. The correct bool flag syntax is `--keep-zip=false`. The existence of `--keep-zip` makes the flag equal to `true`.

**Comment by ptornhult** [comment]
:) Thanks!

**Comment by ptornhult** [comment]
It doesn't seem to work. This is the command we're running and it still keeps the ZIP file after unzipping it:
```
cmd /c %LOCALISE_HOME%\lokalise2 ^
    --token %LOKALISE_TOKEN% ^
    --project-id %LOKALISE_PROJECT% ^
    file download ^
    --format xml ^
    --keep-zip=false ^
    --unzip-to ./locales
```

**Comment by beinarovic** [comment]
Yes, we have confirmed the bug, the problem is only on Windows. We will add it to our tasks.

Thank's for the issue! 😉

**Comment by beinarovic** [comment]
Hey @ptornhult! 

We have fixed this issue in 2.5.2.
